### ISSUE TITLE
tidyjson removed from CRAN

### ISSUE THREAD

**Issue by zapster** [sailthru/tidyjson#60]
It seems like `tidyjson` was removed from CRAN: https://cran.r-project.org/web/packages/tidyjson/
Any hints why that is the case?

**Comment by colearendt** [comment]
Yes - CRAN has an old email address for the maintainer (the package is "orphaned"), and the new version of `dplyr` (0.7.0+) began failing the check, I believe.  Hopeful of having a corrected version on CRAN soonish (of course, it has taken much longer than anticipated).

See https://github.com/jeremystan/tidyjson/issues/104 and https://github.com/jeremystan/tidyjson/issues/97
